dot/network/service.go

```diff
@@ -674,3 +674,13 @@ func (s *Service) Peers() []common.PeerInfo {
 func (s *Service) NodeRoles() byte {
 	return s.cfg.Roles
 }
+
+// HighestBlock returns the highest known block number
+func (s *Service) HighestBlock() int64 {
```

Comment:
any reason why this isn't a `uint64`?

Reply:
I just keep the return type as the type of `syncQueue.currStart`,